specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -4237,6 +4279,40 @@
         }
       }
     },
+    "ConnectorMetadata": {
+      "description": "Gets the list of connectors and run profile names.",
+      "type": "object",
+      "properties": {
+        "connectors": {
+          "description": "The list of connectors.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConnectorMetadataDetails"
+          }
+        },
+        "runProfileNames": {
+          "description": "The list of run profile names.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ConnectorMetadataDetails": {
+      "description": "Details of the connector. Only has ConnectorId & Display Name.",

```

Comment:
[nit] The schema tells us which properties are in this object - it is superfluous in the description.

Reply:
Fixed